packages/core/src/config/recommended.ts

```diff
@@ -25,7 +25,7 @@ export default {
     'operation-operationId-url-safe': 'error',
     'operation-parameters-unique': 'error',
     'operation-tag-defined': 'off',
-    'operation-security-defined': 'error',
+    'security-defined': 'error',
```

Comment:
I'm not sure if this has to be `error`. Public APIs don't have to have a `security` defined in theory. I would change it back to `warn`.

@adamaltman what do you think?

Reply:
It was an error before. Not sure what you mean by a turn back.

Reply:
Yes. I see. I mean how about making it `warn` at all?

Reply:
Let's keep it an error.